android/assets/jsons/translations/German.properties

```diff
@@ -3194,90 +3174,48 @@ Borgholm = Borgholm
- # Requires translation!
-Bischofshofen = 
- # Requires translation!
-Waidhofen = 
- # Requires translation!
-Saalbach = 
- # Requires translation!
-Lienz = 
- # Requires translation!
-Steyr = 
+Austria = Österreich
+Maria Theresa = Maria Theresa
+Shame that it has come this far. But ye wished it so. Next time, be so good, choose your words more wisely.  = Schade, dass es so weit gekommen ist. Aber du hast es so gewollt. Sei das nächste Mal so gut, wähle Deine Worte mit Bedacht.
+What a fool ye are! Ye will end swiftly and miserably. = Was für ein Dummkopf seid ihr! Ihr werdet schnell und elend enden.
+The world is pitiful! There's no beauty in it, no wisdom. I am almost glad to go. = Die Welt ist erbärmlich! Darin liegt keine Schönheit, keine Weisheit. Ich bin fast froh zu gehen.
+The archduchess of Austria welcomes your Eminence to... Oh let's get this over with! I have a luncheon at four o'clock. = Die Erzherzogin von Österreich heißt Eure Eminenz herzlich willkommen... Ach, lasst uns das hinter uns bringen! Ich esse um vier Uhr zu Mittag.
+I see you admire my new damask. Nobody should say that I am an unjust woman. Let's reach an agreement! = I see you admire my new damask. Nobody should say that I am an unjust woman. Let's reach an agreement!
+Oh, it's ye! = Oh, it's ye!
+Diplomatic Marriage = Diplomatische Ehe
+Can spend Gold to annex or puppet a City-State that has been your ally for [amount] turns. = Kann Gold ausgeben, um einen Stadtstaat zu annektieren oder zu Marionetten zu machen, der für [amount] Runden Ihr Verbündeter ist.
+Noble and virtuous Queen Maria Theresa, Holy Roman Empress and sovereign of Austria, the people bow to your gracious will. Following the death of your father King Charles VI, you ascended the thone of Austria during a time of great instability, but the empty coffers and diminished military did litle to dissuade your ambitions. Faced with war almost immediately upon your succession to the thron, you managed to fend off your foes, and in naming your husband Francis Stephen co-ruler, assured your place as Empress of the Holy Roman Empire. During your reigh, you guided Austria on a new path of reform - strengthening the military, replenishing the treasury, and improving the educational system of the kingdom. = Noble and virtuous Queen Maria Theresa, Holy Roman Empress and sovereign of Austria, the people bow to your gracious will. Following the death of your father King Charles VI, you ascended the thone of Austria during a time of great instability, but the empty coffers and diminished military did litle to dissuade your ambitions. Faced with war almost immediately upon your succession to the thron, you managed to fend off your foes, and in naming your husband Francis Stephen co-ruler, assured your place as Empress of the Holy Roman Empire. During your reigh, you guided Austria on a new path of reform - strengthening the military, replenishing the treasury, and improving the educational system of the kingdom.
```

Comment:
Hier fehlt die Übersetzung "Edle und tugendhafte Königin Maria Theresia, Heilige Römische Kaiserin und Herrscherin von Österreich, das Volk beugt sich eurem gnädigen Willen. Nach dem Tod eures Vaters, König Karl VI., habt Ihr den Thron Österreichs in einer Zeit großer Instabilität bestiegen, doch die leeren Staatskassen und das geschrumpfte Militär konnten euren Ehrgeiz nur wenig bremsen. Fast unmittelbar nach eurer Thronbesteigung wurdet Ihr mit einem Krieg konfrontiert. Es gelang euch, eure Feinde abzuwehren, und indem Ihr euren Ehemann Franz Stephan zum Mitregenten ernannt habt, habt Ihr euren Platz als Kaiserin des Heiligen Römischen Reiches gesichert. Während Eurer Regentschaft habt Ihr Österreich auf einen neuen Weg der Reformen geführt - Ihr habt das Militär gestärkt, habt die Staatskasse aufgefüllt und habt das Bildungssystem des Königreichs verbessert." Bitteschön